web/src/components/tasks/CheckElevatorRegistrationStatus.tsx

```diff
@@ -38,8 +40,38 @@ export const CheckElevatorRegistrationStatus = (props: Props): ReactElement => {
     } as Municipality;
   });
 
+  useEffect(() => {
+    if (business?.profileData.communityAffairsAddress?.streetAddress1 && formValues.address1 === "") {
+      formValues.address1 = business.profileData.communityAffairsAddress.streetAddress1;
+    }
+    if (business?.profileData.communityAffairsAddress?.streetAddress2 && formValues.address2 === "") {
+      formValues.address2 = business.profileData.communityAffairsAddress.streetAddress2;
+    }
+    if (business?.profileData.communityAffairsAddress?.municipality && !selectedMunicipality) {
+      setSelectedMunicipality(business.profileData.communityAffairsAddress.municipality);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
[dust] I know this is our pattern, but we should look into fixing this rather than disabling the eslint rule. It might be a good idea for a lunch and learn.

Reply:
Would love to discuss this further in a LnL!